src/main/java/in/oogway/ologger/networkLogger.java

```diff
@@ -0,0 +1,64 @@
+package in.oogway.ologger;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class networkLogger implements oogwayLogger {
+    static Logger networkLogger = LoggerFactory.getLogger(networkLogger.class);
+    Level level;
+
+    public networkLogger(){
+        this.level = Level.WARN;
+    }
+
+    @Override
+    public void setLevel(Level level) {
```

Comment:
You should avoid using same variable names repeatedly. In most cases compiler does scope analysis and knows what you mean but there can be undesirable side effects. And you should avoid variable name duplication.